### PR TITLE
Fix gradle error

### PR DESCRIPTION
**Pull request by kazoni**
Update to openjdk8 - see gradle/gradle#2421

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/java-http-client?pullRequest=45) <br/>All committers have signed the CLA.

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ language: java
 sudo: false
 jdk:
 - oraclejdk8
-- openjdk7
+- openjdk8
```
Comment (mbernier) [inline .travis.yml:5]: @thinkingserious do we need to support the jdk7? That was what was causing the issue here.

**Comment by mbernier** [comment]
Related to #46

**Comment by mbernier** [comment]
Also this: https://docs.gradle.org/current/release-notes.html#java-7-support-is-deprecated

Java7 reached end of life.

Kill it ded!

**Comment by thinkingserious** [comment]
Hello @kazoni,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**